libsolidity/parsing/Token.cpp

```diff
@@ -50,6 +50,44 @@ namespace dev
 namespace solidity
 {
 
+bool ElementaryTypeNameToken::isElementaryTypeName(Token::Value _baseType, string const& _info)
+{
+	if (!Token::isElementaryTypeName(_baseType))
```

Comment:
mmmmm...but now comes the question...where should we do typechecking to make sure that they fit the parameters?